x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/autoscaling/MlMemoryAutoscalingDecider.java

```diff
@@ -185,8 +185,6 @@ public MlMemoryAutoscalingCapacity scale(Settings configuration, AutoscalingDeci
 
         final List<String> partiallyAllocatedModels = mlContext.findPartiallyAllocatedModels();
 
-        // TODO for autoscaling by memory, we only care about if the model is allocated to at least one node (see above)
```

Comment:
I'm thinking we should keep this check in so that we don't downscale memory anyway if there are partially allocated model deployments. What do you think?